tests/jobs/rft/conftest.py

```diff
@@ -41,7 +41,7 @@ def get_mock_data_content_norne():
     return copy.deepcopy(MOCK_DATA_CONTENT_NORNE)
 
 
-def _assert_almost_equal_line_by_line(file1, file2):
+def assert_almost_equal_line_by_line(file1, file2):
```

Comment:
This function can go directly to `test_gedata_rft.py`.

Reply:
yup, and then I will keep the underscore.